includes/fields/class-field-column.php

```diff
@@ -33,7 +33,7 @@ public function render( $field_settings, $form_id, $type = 'post', $post_id = nu
                 <div class="wpuf-column-field-inner-columns">
                     <div class="wpuf-column">
                         <?php while ( $i <= $columns ) { ?>
-                            <div class="<?php echo 'column-' . esc_attr( $i ); ?> <?php echo 'items-of-column-' . esc_attr( $columns ); ?> wpuf-column-inner-fields" style="width: <?php echo esc_attr( $columns_size['column-'.$i] ); ?>; padding-right: <?php echo esc_attr( $column_space ) . 'px'; ?>">
+                          <div class="<?php echo 'column-' . esc_attr( $i ); ?> <?php echo 'items-of-column-' . esc_attr( $columns ); ?> wpuf-column-inner-fields">
```

Comment:
Why is this multiple, seperate <?php echo .... ?> statements? This should just be one concatenated <?php echo ... ?> statement.
`<?php echo 'column-' . esc_attr( $i ) . ' items-of-column-' . esc_attr( $columns ); ?>`